etc/samples/logging.yaml.journald.sample

```diff
@@ -0,0 +1,37 @@
+version: 1
+disable_existing_loggers: False
+formatters:
+    simple:
+        format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        datefmt: "%Y-%m-%dT%H:%M:%S%z"
+
+handlers:
```

Comment:
Would be best to have file handlers mapped to repo/packagename 

eg: bridge_common logs should goto /var/log/tendrl/common/common.log